### PR TITLE
Unbind unregistered keybinds

### LINKED ISSUE #580: Unregistering keybinds will still make then "conflict"
image

### PR DESCRIPTION
**Pull request by Incompleteusern**
Fixes #580

### CODE REVIEW

**Comment by kristofbolyai** [comment]
@Incompleteusern is this a draft because you did not get to test it?

**Comment by Incompleteusern** [comment]
Yes, I have not gotten around to testing it yet

File: common/src/main/java/com/wynntils/core/keybinds/KeyBind.java [hunk 1]
```diff
@@ -43,6 +43,10 @@ public KeyBind(String name, int keyCode, boolean firstPress, Runnable onPress, C
     public KeyBind(String name, int keyCode, InputConstants.Type type, boolean firstPress, Runnable onPress) {
         this.firstPress = firstPress;
         this.keyMapping = new KeyMapping(name, type, keyCode, CATEGORY);
+
+        // Unbound by default
+        keyMapping.setKey(InputConstants.UNKNOWN);
+        KeyMapping.resetMapping();
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/core/keybinds/KeyBind.java]: Why? Don't we want to actually register the keybind? This feels smth like "@StartDisabled" for keybinds. I think the changes in `unregisterKeybind` should be enough

Comment (Incompleteusern) [reply common/src/main/java/com/wynntils/core/keybinds/KeyBind.java]: This should unbind the keybind, and if the keybind is registered it should be bound by the options loading later.

Comment (Incompleteusern) [reply common/src/main/java/com/wynntils/core/keybinds/KeyBind.java]: Actually, registering keybind should bind it too, oops

**Comment by kristofbolyai** [comment]
Technically this still needs testing/fixing? We need to see if KeyMapping constructor does weird behavior

**Comment by kristofbolyai** [comment]
@Incompleteusern Can you prioritize this whenever you are free?

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**